Cleanup duplicate mock in ActivateRecipePanel test

BLoe opened the pull request:
## What does this PR do?

- Follow up to [the comment](https://github.com/pixiebrix/pixiebrix-extension/pull/5233#discussion_r1111265408) on PR #5233

twschiller commented:
@BLoe thanks for the PR: can you clean up the related comment here? https://github.com/pixiebrix/pixiebrix-extension/pull/5233/files#diff-6c4913b2c4e160726451842d23ac0c9b495c6fd315b543212ebcd1cd224fcfa4R182

Just glancing at GitHub, I'm still now sure how the test works. There's something causing it to default to not set?

BLoe commented:
> Just glancing at GitHub, I'm still now sure how the test works. There's something causing it to default to not set?

I accidentally put two `jest.mock()` calls in there for the same hook, `useQuickbarShortcut()`. The first call was mocking the return value to `{ isConfigured: true }`, and I removed that one. There's another call to mock this hook that's still in the test, now on line 90:
![image](https://user-images.githubusercontent.com/2801308/220469874-6f3ed0c0-397c-4073-9826-4d5cbef04597.png)

Since this call was _after_ the previous mock call that I removed, it was overriding the mock and returning `false` for `isConfigured`.
